evalml/preprocessing/data_splitters/balanced_classification_sampler.py

```diff
@@ -10,12 +10,14 @@
 class BalancedClassificationSampler(SamplerBase):
     """Class for balanced classification downsampler."""
 
-    def __init__(self, sampling_ratio=0.25, min_samples=100, min_percentage=0.1, random_seed=0):
+    def __init__(self, sampling_ratio=0.25, sampling_ratio_dict=None, min_samples=100, min_percentage=0.1, random_seed=0):
         """
         Arguments:
             sampling_ratio (float): The smallest minority:majority ratio that is accepted as 'balanced'. For instance, a 1:4 ratio would be
                 represented as 0.25, while a 1:1 ratio is 1.0. Must be between 0 and 1, inclusive. Defaults to 0.25.
 
+            sampling_ratio_dict (dict): A dictionary specifying the desired balanced ratio for each target value. Defaults to None.
```

Comment:
I think it's important to specify that this overrides `sampling_ratio` if both are set?

Reply:
Good call, adding!